doc/lib_num_to_formatted_str.md

```diff
@@ -74,16 +74,18 @@ Description:
 ### time_formatting
 
 Args:
-  1. A number (scaler), The number be formatted
-  2. A number (scaler), Selects type of format to be used, Range from 0 to 6
+  1. A number (scaler), timeSec     The number be formatted
+  2. A number (scaler), formatType  Selects type of format to be used, Range from 0 to 6
       * Defaulted to 0
-  3. A number (scaler), The rounding for the seconds place, Range from 0 to 2
+  3. A number (scaler), rounding    The rounding for the seconds place, Range from 0 to 2
       * Defaulted to 0
-  4. A boolean,         If True the return uses "T+" or "T-" to denote positive or negative else it will use " " or "-"
+  4. A boolean,         prependT    If True prepends a "T+ " or "T- " to the format ("T- " or "T  " if showPlus is False)
```

Comment:
This is not correct because the code as it is will ignore `showPlus` if `prependT` is true.

Reply:
Agreed. I corrected the behavior in c72fd3c.